### PR TITLE
new gradient on item detail

### PR DESCRIPTION
**Pull request by MathiasOhrstrom**

### CODE REVIEW

File: app/src/main/res/drawable/ic_back_white.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (hedvigoscar) [inline app/src/main/res/drawable/ic_back_white.xml]: Is this file necessary? Can we achieve the same with an icon tint?

File: app/src/main/res/drawable/key_gear_item_detail_photo_gradient.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <gradient
+        android:startColor="#73000000"
+        android:endColor="#00FFFFFF"
```
Comment (hedvigoscar) [inline app/src/main/res/drawable/key_gear_item_detail_photo_gradient.xml]: Should this gradient be white?

File: app/src/main/res/layout/key_gear_item_detail_photo.xml [hunk 3]
```diff
@@ -17,9 +17,8 @@
 
     <ImageView
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
+        android:layout_height="300dp"
```
Comment (hedvigoscar) [inline app/src/main/res/layout/key_gear_item_detail_photo.xml]: dimensify

File: app/src/main/res/values-v29/styles.xml [hunk 4]
```diff
@@ -12,5 +12,6 @@
     <style name="Hedvig.Theme.KeyGear">
         <item name="android:navigationBarColor">@android:color/transparent</item>
         <item name="android:statusBarColor">@android:color/transparent</item>
+        <item name="android:windowLightStatusBar">false</item>
```
Comment (hedvigoscar) [inline app/src/main/res/values-v29/styles.xml]: `Hedvig.Theme.KeyGear` is used by other screens, does this interfere with their looks?

File: app/src/main/res/values/colors.xml [hunk 5]
```diff
@@ -39,4 +39,5 @@
     <color name="translucent_dark_mode">#aa0a0d14</color>
     <color name="dark_mode_elevation_1">#24272e</color>
     <color name="dark_mode_semi_transparent">#e624272e</color>
+    <color name="translucent_tool_bar">#730A0D14</color>
```
Comment (hedvigoscar) [inline app/src/main/res/values/colors.xml]: How does this colour differ from `translucent_dark_mode` (the one we used before), and should we be using that one instead?

File: app/src/main/java/com/hedvig/app/feature/keygear/ui/createitem/CreateKeyGearItemActivity.kt [hunk 6]
```diff
@@ -150,7 +149,7 @@ class CreateKeyGearItemActivity : BaseActivity(R.layout.activity_create_key_gear
             interpolator = AccelerateDecelerateInterpolator()
             duration = SAVE_BUTTON_TRANSITION_DURATION
             addUpdateListener { va ->
-                saveContainer.updateLayoutParams<CoordinatorLayout.LayoutParams> {
+                saveContainer.updateLayoutParams<FrameLayout.LayoutParams> {
```
Comment (hedvigoscar) [inline app/src/main/java/com/hedvig/app/feature/keygear/ui/createitem/CreateKeyGearItemActivity.kt]: I think this change might go away if you merge the parent feature 🙂

File: app/src/main/res/values-v29/styles.xml [hunk 7]
```diff
@@ -13,4 +13,8 @@
         <item name="android:navigationBarColor">@android:color/transparent</item>
         <item name="android:statusBarColor">@android:color/transparent</item>
     </style>
+
+    <style name="Hedvig.Theme.KeyGear.ItemDetail" parent="Hedvig.Theme.KeyGear">
```
Comment (hedvigoscar) [inline app/src/main/res/values-v29/styles.xml]:
```suggestion
    <style name="Hedvig.Theme.KeyGear.ItemDetail">
```


`name="Hedvig.Theme.KeyGear.ItemDetail"` actually implies `parent="Hedvig.Theme.KeyGear"` 🙂

Comment (MathiasOhrstrom) [reply app/src/main/res/values-v29/styles.xml]: aaahh, smart
